scripts/jest/setupFixture.ts

```diff
@@ -1,31 +1,26 @@
 import path from "path";
-import findUp from "find-up";
 import fs from "fs-extra";
 import tmp from "tmp";
-import { init, stageAndCommit, gitFailFast } from "../git";
```

Comment:
This helper file was previously in the workspace-tools package, but I moved it to scripts (along with the fixtures) so it can be used by multiple packages. To avoid circular deps, I had to remove usage of some utilities and types.